hokusai/production.yml

```diff
@@ -173,4 +173,35 @@ spec:
                   - key: tier
                     operator: In
                     values:
-                    - background
\ No newline at end of file
+                      - background
+
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: positron-data-sync
+spec:
+  schedule: "*/10 * * * *"
```

Comment:
Not familiar with this syntax, what is the frequency of these jobs?

Reply:
Yeah, so right now this is every 10 mins. I typically use [cronguru](https://crontab.guru/#*/10_*_*_*_*) to build and decode crontab schedules.

Reply:
I'm hoping that we can move all the staging-dependent processes into production.  If we did it daily that might actually help for the content sales process.  Is there a standard for how frequently this will happen for other db's?

Reply:
> At this point we only need to grab screencaps of data in production, so syncing daily might actually make that process easy -- just wait till the sync finishes or manually update in staging before grabbing a screenshot.

Not really following you here.

>  Is there a standard for how frequently this will happen for other db's?

I don't believe there's a standard yet. FWIW, it's very easy for us to tweak and iterate on the cron schedule, so we just need to land on our first attempt here. Maybe every evening at 10 PM ET considering your previous comment?

Reply:
that sounds like a good schedule to me -- i just meant that if the sync happens regularly enough in production, they can wait for a sync to staging, and publish the content there before grabbing their screengrabs.